Integrations/integration-ServiceNow.yml

```diff
@@ -3421,7 +3479,37 @@ script:
       description: Table system name, e.g Asset
       type: string
     description: Get table names by a label to use in commands
+  - name: servicenow-get-ticket-notes
+    arguments:
+    - name: id
+      required: true
+      default: true
+      description: Ticket System ID
+    - name: limit
+      description: Limit for the ticket notes
+      defaultValue: "10"
+    - name: offset
+      description: Offset of the ticket notes
+      defaultValue: "0"
+    outputs:
+    - contextPath: ServiceNow.Ticket.ID
+      description: Ticket ID
+      type: string
+    - contextPath: ServiceNow.Ticket.Note.Value
+      description: Ticket note value
+      type: unknown
+    - contextPath: ServiceNow.Ticket.Note.CreatedOn
+      description: Ticket note created on
+      type: date
+    - contextPath: ServiceNow.Ticket.Note.CreatedBy
+      description: Ticket note created by
+      type: string
+    - contextPath: ServiceNow.Ticket.Note.Type
+      description: Ticket note type
+    description: Get notes from the specified ServiceNow ticket - Permissions are
```

Comment:
which permissions are required?
what is the error returned in case the user does not have the required permissions?